pkg/cmd/cmd.go

```diff
@@ -4,16 +4,17 @@ package cmd
 
 import (
 	"fmt"
+	"math/big"
+
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/keep-network/keep-common/pkg/chain/ethereum"
 	"github.com/keep-network/keep-common/pkg/cmd/flag"
-	"github.com/urfave/cli"
+	"github.com/spf13/cobra"
 )
 
 const (
-	blockFlag        string = "block"
-	blockShort       string = "b"
-	transactionFlag  string = "transaction"
-	transactionShort string = "t"
+	blockFlag  string = "block"
+	blockShort string = "b"
 	// SubmitFlag allows for urfave/cli definition and lookup of a boolean
```

Comment:
Since we removed the `urfave/cli` package, this command should be updated, so that it does not mention it.

Reply:
fd26c8128e2c05b4ef7a406e5d161eed7f0f77e5